terminate sync docs (#7479)
harness/developer-hub: docs/continuous-delivery/gitops/use-gitops/sync-gitops-applications.md

Here is how the resources would look in Harness after the sync process is complete.

![](./static/harness-git-ops-application-set-tutorial-40.png)

> ## Terminate Sync

Reviewer: Terminate sync

> ## Terminate Sync

Author: ```suggestion
## Terminate sync
```

> You can terminate an in-progress sync. To do so, go to the application page for the syncing app and find the **Terminate Sync** button in the top right corner of the UI, replacing the **Sync** button when a sync is in progress. 

Reviewer: To terminate an in-progress sync, go to the application for the syncing app and locate the **Terminate Sync** button in the top right corner of the UI. Replace the **Sync** button when a sync is in progress.

> You can terminate an in-progress sync. To do so, go to the application page for the syncing app and find the **Terminate Sync** button in the top right corner of the UI, replacing the **Sync** button when a sync is in progress. 

Author: ```suggestion
To terminate an in-progress sync, go to the application for the syncing app and locate the **Terminate Sync** button in the top right corner of the UI. Replace the **Sync** button when a sync is in progress.
```
